cmd/generate.go

```diff
@@ -167,6 +167,24 @@ function results_dir() {
   echo "$logdir/$1.$(date +%Y%m%d.%T)"
 }
 
+function check_empty_result() {
+  # There is a random roachprod issue that we recently identified:
+  # After a test finished successfully in a host, the following "roachprod get"
```

Comment:
That seems like a serious bug; do we know the root cause of why `get` fails to copy the contents _and_ doesn't report an error?

Reply:
This happened randomly, out of 200 tests that I had completed, at least two dozens failed due to this issue. Another interesting observation is that almost all failures happened in group way, i.e. when tpcc failed on one machine type, the cpu test also failed, vice versa.  I couldn't see any pattern that could consistently repro. This probably is related to the pattern I run a machine type's test, I always created one cluster, then run both tpcc and cpu one after another. I am guessing it somehow due to multiple parallel tests running in a region, causing some network bandwidth issue. that the `scp` op, which the `roachprod get` cmd uses to copy failed. Another possibility might be related with the recent change of `LimitNOFILE` from unlimited to 65536. Before the rerun of failed test cases due to this, I increased it to 655360, so far, not a single rerun failure happened again.

Reply:
@miretskiy good suggestion. Added retry logic. PTAL

Reply:
>Another possibility might be related with the recent change of LimitNOFILE from unlimited to 65536.

It wasn't that recent if we're talking about the same config.: https://cockroachlabs.slack.com/archives/C017HTR8L48/p1640301229447300?thread_ts=1640200237.395900&cid=C017HTR8L48

> Before the rerun of failed test cases due to this, I increased it to 655360, so far, not a single rerun failure happened again.

Is this change checked in somewhere?

Reply:
The reason we couldn't identify this in test run stage is because we seldom kicked off multiple parallel runs then, even with parallel run these two issues that are affected by too small file descriptor limit only happened randomly, only when we kicked off first official run when we always run parallel, this began to surface. At that time Nathan thought it was because of the limit is set too high, nobody realized it's because of this file limit change.
I already submitted a PR, include the file owner and you as reviewer.